app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/spaces/space_parent_badge.dart';
+import 'package:acter/features/chat/providers/chat_providers.dart';
+import 'package:acter/features/chat/widgets/member_list.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class RoomProfilePage extends ConsumerWidget {
+  const RoomProfilePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final room = ref.watch(currentConvoProvider)!;
+    final members =
+        ref.watch(chatMembersProvider(room.getRoomIdStr())).requireValue;
+    final myMembership =
+        ref.watch(spaceMembershipProvider(room.getRoomIdStr()));
+    final List<Widget> topMenu = [
+      Text(
+        'Members (${members.length})',
+        style: Theme.of(context).textTheme.titleMedium,
+      ),
+    ];
+    if (myMembership.hasValue) {
+      final membership = myMembership.value!;
+      if (membership.canString('CanInvite')) {
+        topMenu.add(
+          IconButton(
+            icon: Icon(
+              Atlas.plus_circle_thin,
+              color: Theme.of(context).colorScheme.neutral5,
+            ),
+            iconSize: 28,
+            color: Theme.of(context).colorScheme.surface,
+            onPressed: () => context.pushNamed(
+              Routes.spaceInvite.name,
+              pathParameters: {'spaceId': room.getRoomIdStr()},
+            ),
+          ),
+        );
+      }
+    }
+
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          SliverAppBar(
+            elevation: 0.0,
+            actions: <Widget>[
+              PopupMenuButton<int>(
+                icon: const Icon(Atlas.dots_vertical, size: 22),
+                itemBuilder: (context) => [
+                  PopupMenuItem<int>(
+                    value: 0,
+                    child: Row(
```

Comment:
am I missing something or does this action not have nothing it does when clicked? Then let's either remove that section or hook it up to the `room.leave()`.